client/app/ui-components/tile/custom/item/cui-tile-item-status.component.js

```diff
@@ -3,24 +3,68 @@ angular.module("managerApp")
         replace: true,
         restrict: "E",
         controller: class CuiTileItemStatusCtrl {
+            constructor ($element) {
+                this.$element = $element;
+            }
+
+            $postLink () {
+                this.$element.addClass("cui-tile__item");
+            }
         },
         scope: true,
         controllerAs: "$ctrl",
+        transclude: true,
         template: `
-            <div data-ng-class="{ 'cui-dropdown-menu-container': $ctrl.actions }" class="cui-tile__item cui-tile-item-status">
-                <div class="cui-tile__definition cui-tile-item-status__definition">
-                    <cui-status-icon data-type="{{ $ctrl.type }}"></cui-status-icon>
-                    <dl class="cui-tile-item-status__description">
-                        <cui-tile-definition-term data-term="$ctrl.term"></cui-tile-definition-term>
-                        <cui-tile-definition-description data-description="$ctrl.description"></cui-tile-definition-description>
-                    </dl>
-                </div>
-                <cui-tile-action-menu data-ng-if="$ctrl.actions" data-actions="$ctrl.actions"></cui-tile-action-menu>
+            <div class="cui-tile-item-status" data-ng-class="{ 'cui-dropdown-menu-container': $ctrl.actions }">
+                <div class="d-flex">
+                    <div class="cui-tile__item-main" data-ng-if="$ctrl.term">
+                        <cui-tile-definitions data-ng-if="$ctrl.term">
+                            <cui-tile-term-status data-term="$ctrl.term"></cui-tile-term-status>
+                            <cui-tile-description-status data-type="{{ $ctrl.type }}" 
+                                data-description="$ctrl.description"></cui-tile-description-status>
+                        </cui-tile-definitions>
+                    </div>
+                    <div class="cui-tile__item-main" data-ng-if="!$ctrl.term && !$ctrl.description" data-ng-transclude></div>
+                    <cui-tile-action-menu data-ng-if="$ctrl.actions" data-actions="$ctrl.actions"></cui-tile-action-menu>
+                </div> 
             </div>`,
         bindToController: {
             term: "<",
             description: "<",
             actions: "<",
             type: "@"
         }
+    }))
+    .directive("cuiTileTermStatus", () => ({
+        replace: true,
+        restrict: "E",
+        controllerAs: "$ctrl",
+        controller: class CuiTileDefinitionTermCtrl {},
```

Comment:
you can remove class name just : class {},
this create an anonyme class

Reply:
Done